webproject/src/test/java/com/google/sps/util/HtmlParserTest.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.util;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+/**
+ * Test class for HtmlParser.
+ */
+public class HtmlParserTest {
+    private static final String HTML_STUB = "<!DOCTYPE html>\n<head></head><body><div>Stub</div></body></html>";
+    private static final String FILE_NAME = "testFile.html";
+
+    // This must be public
+    @ClassRule
+    public static TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    @Test
+    public void parseHtmlFromFile_returnsParsedHtml() {
```

Comment:
you can put test as return Exception. this way you dont need try catch block on every test.
throwing Exception on test is preferrable since you dont want to have an exception in your test go silently, since the use of test is to catch it

Reply:
Done.